### DISCUSSION TITLE
Using 1.0 for our dapp post March 1st 2023

### DISCUSSION THREAD

**Discussion by 340rahul** [WalletConnect/walletconnect-monorepo#1840] [Developer Support]
Hello WC!

Congratulations on the v2 release. I had a question regarding our react native dapp. We are currently using walletconnect v1.0 which is working pretty well for us, and since v2 doesn't officially support react-native yet, we were thinking of sticking to v1.0 for a few more months. Would we still be able to do that post the March 1st deadline? Will the bridge servers go down as well?

**Comment by crypblizz8** [comment]
Hey there, we will be sunsetting from March 1 so strongly recommend moving over before then. 

We have some starter code at [react-native examples](https://github.com/WalletConnect/react-native-examples). We will be focusing on getting as much documentation / expanding the codebase in the coming weeks.

Hope that helps.

**Comment by marcelarutkowska** [comment]
Hi @crypblizz8, could you be more specific, please? Does it mean that v1 will stop working or not?
